GabrielaRamirez/EjerciciosHackerrank/src/challenge/Nieve.java

```diff
@@ -0,0 +1,10 @@
+package challenge;
+
+public class Nieve extends Icecream{
+
+    @Override
+    public void printFlavour(String flavour){
+        System.out.println("Nieve de " + flavour);
```

Comment:
English please

Reply:
:D ok

Reply:
Is Spanglish 'cause by one side is IceCream and the other is Nieve 😆  (I'm kidding)

Reply:
😆